scalikejdbc-interpolation/src/test/scala/scalikejdbc/QueryInterfaceSpec.scala

```diff
@@ -353,7 +353,7 @@ class QueryInterfaceSpec extends FlatSpec with Matchers with DBSettings with SQL
         {
           val notInClauseResults = withSQL {
             select.from(Order as o)
-              .where.not.in(o.id, Seq())
+              .where.not.in(o.id, Seq[Int]())
```

Comment:
My suggestion here is just a nice-to-have thing. Ignore it if there is no solution for now.